guider/guider

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Why did you restrict the shell to bash?
It will cause error on systems without bash.
There are also various shells such as dochsh, dash, rbash, rsh...